stepflow-rs/crates/stepflow-core/src/values/value_resolver.rs

```diff
@@ -170,9 +172,12 @@ impl<L: ValueLoader> ValueResolver<L> {
         let path_result = if let Some(path) = expr.path() {
             match base_result {
                 FlowResult::Success { result } => {
+                    tracing::debug!("Resolving path '{}' on value: {:?}", path, result.as_ref());
```

Comment:
Ditto -- generally prefer putting the references in the string -- `"Resolving path '{path}' on value: {result:?}"`.